.github/workflows/sonarcloud.yaml

```diff
@@ -0,0 +1,51 @@
+name: Sonarcloud Scan
+
+on:
+  push:
```

Comment:
following from the previous comment, this will run too often.
- can this run on a schedule (ex. weekly)?
- only run this on the main alluxio/alluxio fork. we don't want other people's forks to run this job.
- only on master branch. i don't know how the receiving server will handle branches, but we also don't care much about the analysis for previous branches

Reply:
Added schedule time

Reply:
- remove the `push:` block
- only run the cron job on master
- only run the job on the alluxio/alluxio fork

Reply:
done

Reply:
i do not think it is done...